src/workloads/query/AggregationsOutput.yml

```diff
@@ -0,0 +1,520 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This test exercises both $out and $merge aggregation stages. It does this based on all combinations of the following:
```

Comment:
Subjectively, it looks like workloads that write to disk are more volatile. Is it possible to measure how "noisy" this workload is and if we are actually getting some useful info from it?

Reply:
I'll run it a few times to see the variance

Reply:
So the variance isn't bad but it's also not great. I've seen a throughput variance of about 10%-ish at worst in 4 sys-perf runs ([1](https://spruce.mongodb.com/version/659d70c261837d09e1d804e5/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC), [2](https://spruce.mongodb.com/version/659d70c7d1fe07059e2479e4/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC), [3](https://spruce.mongodb.com/version/659d70cc0305b903976733f3/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC), [4](https://spruce.mongodb.com/version/659d70d11e2d170d8d558d77/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)).

I see here two paths for tackling the potential disk noise:
* We add another DSI variant that is ephemeral as I haven't seen any in the list
* We ignore the noise issue altogether since it is part of the WT life and AWS disk variance itself

I would argue that doing the latter is the best path here since doing the former is not a reflection of what the end-user would see as it is a completely different storage engine (RAM vs disk semantics).